.github/workflows/test-suite.yml

```
@@ -154,3 +158,31 @@ jobs:
+      with:
+        default: 3.12
+        command: pip install -U pip  # upgrade pip after installing python
+
+    - name: Install dependencies
+      run: |
+        pip install -r requirements.txt -r requirements-dev.txt
+    
+    - name: Configure to use REDIS via localhost
+      run: |
+        sed -i "s/localhost/redis/g" config/vsm/app.json
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)